s2spy/_plot.py

```diff
@@ -160,16 +168,17 @@ def matplotlib_visualization(
                         is aligned by the anchor date, so that all anchor years line up
                         vertically.
         add_length: If the length of every periods should be displayed. Defaults False.
+        add_yticklabels: If the years should be displayed on the y-axis ticks.
     """
-    _, ax = plt.subplots()
+    fig, ax = plt.subplots(figsize=(8,5))
```

Comment:
Still a bit cramped: 

![output](https://user-images.githubusercontent.com/17080502/201101549-448029b1-5c40-4a87-9a0d-65acc50a7cd3.png)

Legend takes up a lot of space, and the axis has an unnecessarily large extent. Details though, don't spend too long on it.

Reply:
I like that last proposal. How necessary is the legend anyway? Only for first time users I'd say. Perhaps make it optional?